### ISSUE TITLE
Documentation: repos.getContent should be repos.getContents

### ISSUE THREAD

**Issue by sampart** [octokit/plugin-rest-endpoint-methods.js#151]
https://octokit.github.io/rest.js/v18#repos-get-content includes the following code example:

```
octokit.repos.getContent({
  owner,
  repo,
  path,
});
```

However, the method is actually called `getContents` (with an `s`):

![Screenshot 2020-06-26 at 15 31 23](https://user-images.githubusercontent.com/1507328/85868408-20cbff80-b7c2-11ea-9316-29914a4e6173.png)

**Comment by gr2m** [comment]
`repos.getContent` is correct, it was recently renamed in [v4.0.0](https://github.com/octokit/plugin-rest-endpoint-methods.js/releases/tag/v4.0.0), and it was deprecated in the latest v3 releases. Here is the published code for v4.0.0:
https://unpkg.com/browse/@octokit/plugin-rest-endpoint-methods@4.0.0/dist-types/generated/method-types.d.ts#L4601

It looks like you have an older version? Can you run `npm ls @octokit/plugin-rest-endpoint-methods`?

**Comment by sampart** [comment]
Hi @gr2m, thanks for the info.  Yes, it does look like I'm running an older version, so that would explain it:

```
$ npm ls @octokit/plugin-rest-endpoint-methods             
codeql@0.0.0 /Users/sam/code/codeql-action
└─┬ UNMET DEPENDENCY @octokit/rest@17.1.0
  └── UNMET DEPENDENCY @octokit/plugin-rest-endpoint-methods@3.3.1 

npm ERR! missing: @octokit/rest@17.1.0, required by codeql@0.0.0
npm ERR! missing: @octokit/plugin-rest-endpoint-methods@3.3.1, required by @octokit/rest@17.1.0

```

I'm not sure why it claims that the deps are missing, but the version numbers do show it's an older version.
